docs/htdocs/info/docs/tools/vep/script/vep_cache.html

```diff
@@ -38,7 +38,13 @@ <h1 id="top"><span style="color:#006;padding-right:15px">Variant Effect Predicto
 
   <ul>
     <li><b><a href="#cache">Cache</a></b> - a downloadable file containing all transcript models, regulatory features and variant data for a species</li>
-    <li><b><a href="#gff">GFF or GTF</a></b> - use transcript models defined in a tabix-indexed GFF or GTF file</li>
+    <li><b><a href="#gff">GFF or GTF</a></b> - use transcript models defined in a tabix-indexed GFF or GTF file
+      <ul><li>
+        Requires a <b><a href="vep_cache.html#fasta">FASTA</a></b> file in
+        offline mode or if the desired species or assembly is not part of the
```

Comment:
A link to `--offline` mode is better here - 

```suggestion
        <a href="vep_options.html#opt_offline">--offline</a> mode or if the desired species or assembly is not part of the
```